common/djangoapps/enrollment/handlers.py

```diff
@@ -0,0 +1,67 @@
+"""
+Handlers and actions related to enrollment.
+"""
+import logging
+
+from edxmako.shortcuts import render_to_string
+from django.conf import settings
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+from django.core.urlresolvers import reverse
+from django.core.mail import send_mail
+from django.dispatch import receiver
+
+from student.models import ENROLL_STATUS_CHANGE, EnrollStatusChange
+
+log = logging.getLogger(__name__)
+
+
+@receiver(ENROLL_STATUS_CHANGE)
+def send_email_to_staff_on_student_enrollment(sender, event=None, user=None, **kwargs):  # pylint: disable=unused-argument
+    """
+    Sends an e-mail to staff after a new enrollment.
+    This feature can be enabled by setting the e-mail in ENROLLMENT_NOTIFICATION_EMAIL in lms.env.json, or by using
+    a SiteConfiguration variable of the same name. Disabled by default.
+    """
+
+    if event == EnrollStatusChange.enroll:
+        # If there's a SiteConfiguration variable (including ''), we use it instead of the lms.env.json setting
```

Comment:
Nit: Does this really warrant a separate comment? This pattern is used throughout the codebase *a lot*.